class.jetpack.php

```diff
@@ -2782,7 +2787,7 @@ public static function activate_default_modules(
 			}
 
 			ob_start();
-			require $file;
+			require_once $file;
```

Comment:
Just dropping a note that this is here because when authorizing a user via CLI I was seeing an error due to requiring a file twice.